src/App.js

```diff
@@ -1,16 +1,29 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog.js';
+import { useState } from 'react';
 
 const App = () => {
+  const [likesCount, setLikesCount] = useState(0);
+  const updateLikes = (isLike) => {
+    if (isLike) {
+      setLikesCount(likesCount + 1);
+    } else {
+      setLikesCount(likesCount - 1);
+    }
+  };
```

Comment:
This is a good start, but we would definitely want a second function that updates the state of the messages that you have created. Right now, you have no way of adequately updating the messages themselves. The heart buttons will change because you have taken care of that within your chatEntry components, but you have nothing connecting those components with the chatMessages you collected earlier. As a result, they will not change.